dockerplugin.py

```diff
@@ -411,20 +411,28 @@ def configure_callback(self, conf):
 
     def init_callback(self):
         self.client = docker.Client(
-                base_url=self.docker_url,
-                version=DockerPlugin.MIN_DOCKER_API_VERSION)
+            base_url=self.docker_url,
+            version=DockerPlugin.MIN_DOCKER_API_VERSION)
         self.client.timeout = self.timeout
 
-        # Check API version for stats endpoint support.
         try:
             version = self.client.version()['ApiVersion']
+        except IOError, e:
+            log.exception(('Unable to access Docker daemon at {url} '
+                           'This may indicate SELinux problems. : {error}')
+                          .format(url=self.docker_url,
+                                  error=e))
+            return False
+
+        # Check API version for stats endpoint support.
+        try:
             if StrictVersion(version) < \
                     StrictVersion(DockerPlugin.MIN_DOCKER_API_VERSION):
                 raise Exception
```

Comment:
You don't really need a try/except block here anymore, just do `if/log/return False`